src/ApplicationInsights.Kubernetes/Utilities/SDKVersionUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Reflection;
+
+namespace Microsoft.ApplicationInsights.Kubernetes.Utilities
+{
+    internal sealed class SDKVersionUtils
+    {
+        #region Signleton
+        private SDKVersionUtils() { }
+        private static readonly SDKVersionUtils _instance = new SDKVersionUtils();
+        public static SDKVersionUtils Instance => _instance;
```

Comment:
Why not just remove _instance and make Instance a readonly field?

Reply:
When a member is public, it is a property than a field. I can list the reasons but I don't want to run into the ever long-haul debates here.